Sources/Paywall/Store/Store.swift

```diff
@@ -73,9 +75,35 @@ class Store {
     
 	func add(config: ConfigResponse) {
 		var data = [String: Bool]()
-		config.triggers.forEach { data[$0.eventName] = true }
-		cache.write(dictionary: data, forKey: "store.config")
+        config.triggers.filter({ (trigger) in
+            switch(trigger.triggerVersion) {
+            case .V1:
+                return  true
+            default:
+                return false
+            }
+        }).forEach { data[$0.eventName] = true }
+
+        let v2TriggersArray: [TriggerV2?] = config.triggers.map { (trigger) in
+            switch(trigger.triggerVersion) {
+            case .V1:
+                return nil
+            case .V2(let triggerV2):
+                return triggerV2
+            }
+        }.filter {
+            (triggerOrNil) in
+            return triggerOrNil != nil ? true : false
+        }
+        
+        self.v2Triggers = v2TriggersArray.reduce([String:TriggerV2](), { (result, trigger) in
+            var result = result
+            result[trigger!.eventName] = trigger!
+            return result
+        })
+        cache.write(dictionary: data, forKey: "store.config")
```

Comment:
Right now data is only the eventNames from the v1 triggers. Wasn't sure about the syntax so decided to just skip it for now.